lighthouse-core/audits/script-treemap-data.js

```diff
@@ -166,26 +167,31 @@ class ScriptTreemapDataAudit extends Audit {
     /** @type {LH.Treemap.Node[]} */
     const nodes = [];
 
-    let inlineScriptLength = 0;
+    // Group inline scripts by URL.
+    // At most, the treemap will have 1 inline node for each frame.
+    /** @type {Map<string, number>} */
+    const inlineNodeLengths = new Map();
+
     for (const script of artifacts.Scripts) {
-      // Combine so that inline scripts show up as a single root node.
-      if (script.url === artifacts.URL.finalUrl) {
-        inlineScriptLength += (script.content || '').length;
-      }
+      if (!isInline(script)) continue;
+      let length = inlineNodeLengths.get(script.url) || 0;
+      length += script.length || 0;
+      inlineNodeLengths.set(script.url, length);
```

Comment:
Can you undo these changes? I am reworking inline scripts here: https://github.com/GoogleChrome/lighthouse/pull/13802/

Reply:
Honestly the treemap stuff is why this is a separate PR from #13793. I'll just close and move the unminified js stuff over there.